src/ints/bios.cpp

```diff
@@ -1211,14 +1212,25 @@ class BIOS final : public Module_base{
 		/* INT 12 Memory Size default at 640 kb */
 		callback[2].Install(&INT12_Handler,CB_IRET,"Int 12 Memory");
 		callback[2].Set_RealVec(0x12);
-		if (IS_TANDY_ARCH) {
+		if (machine == MCH_TANDY) {
```

Comment:
Nice! I too dislike the macro hackery...

Reply:
`IS_TANDY_ARCH` actually means `machine == MCH_TANDY || machine == MCH_PCJR`.  I did this because I was adding further distinctions between the two but it's maybe badly named.